presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -35,10 +35,11 @@
 public final class Standard
         implements EnvironmentExtender
 {
-    public static final String CONTAINER_PRESTO_ETC = "/docker/presto-product-tests/conf/presto/etc";
+    public static final String CONTAINER_BASE_PATH = "/docker/presto-product-tests";
```

Comment:
This doesn't really belong here, to this class.
It's a convention of the module, not specific to this `EnvironmentExtender`
For now, i'd inline this back. We may find a good place for this later.